### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by ghost**
General housekeeping PR, notable changes:

- fix import path alias
- reorganize theme files
- tidy up config files
- ~re-enable eslint on build~

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
-  // TODO: remove eslint ignore until all sources are properly linted
+  // https://nextjs.org/docs/api-reference/next.config.js/ignoring-eslint
   eslint: {
-    ignoreDuringBuilds: true
+    ignoreDuringBuilds: false
```
Comment (pveyes) [inline next.config.js]: I thought this is `true` because of we already have GH action for lint #28

Comment (ghost) [reply next.config.js]: Ah you're right, let me revert that. 🙏🏻

Comment (ghost) [reply next.config.js]: Reverted on 59ee8031404bc2c24e0fc167476875d760c9cb35 with comments describing reason disabling ESLint on build.

**Review by pveyes — APPROVED** [review]
LGTM

File: src/theme/index.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import breakpoints from './breakpoints';
+import icons from './icons';
+
+import { extendTheme, ThemeOverride } from '@chakra-ui/react';
+
+export default extendTheme(<ThemeOverride>{
+  breakpoints,
+
+  colors: {
+    black: '#16161D',
```
Comment (pveyes) [inline src/theme/index.ts]: my ocd brain is tingling

```suggestion
    black: '#16161d',
```

**Comment by pveyes** [comment]
tested locally LGTM, merging this

**Pull request merged**